content/en/docs/instrumentation/java/getting-started.md

```diff
@@ -49,7 +58,7 @@ dependencies {
 }
 ```
 
-### Code
+### Create the sample HTTP Server
```

Comment:
How about:

```suggestion
### Create and launch an HTTP Server
```

If you prefer the original title, then maybe at least replace "sample" with "simple"?